components/dashboard/context/DashboardContext.tsx

```
@@ -0,0 +1,147 @@
+
+// Create the context with a default value
+const DashboardContext = createContext<DashboardContextType | undefined>(undefined);
+
+export function DashboardProvider({ children }: { children: ReactNode }) {
+  const { user, isLoading: userLoading } = useUser();
+  const { isProvider, isLoading: providerLoading } = useProvider();
+  
+  // Dashboard state
+  const [isSidebarOpen, setIsSidebarOpen] = useState(true);
+  const [preferences, setPreferences] = useState<DashboardPreferences>(defaultPreferences);
+  const [recentActivity, setRecentActivity] = useState<any[]>([]);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)